infor-design/enterprise
# Misc - Fix Several Locale issues

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This pr fixes a few locale issues.
- added new translation strings
- added support for setting nb and nn and maping it to no locale
- made the examples for date oriented components (calendar, datepicker, timepicker ect) - work with setting the language and having it still translate day/month names
- corrected some issues in the RU locale

**Related github/jira issue (required)**:
Fixes #3468 
Fixes #3455 
Fixes #3244 
Fixes #3464 

**Steps necessary to review your pull request (required)**:
**3468**
- go to http://localhost:4000/components/datepicker/example-timeformat.html?locale=nb-NO
- when the picker is open should be a : between the time fielss
- pick a date in the first field
- should be a time with dots in the date annd colon in the time (11.02.2020 00:00)

**3455**
- go to a page like http://localhost:4000/components/about/example-index
- in the console type `Locale.setLanguage('nb')
- should be no erro
- open the about and it will show norwegian
- reload (or not)
- in the console type `Locale.setLanguage('nn')
- should be no error
- open the about and it will show norwegian

**3244**
- go to  http://localhost:4000/components/calendar/test-specific-lang.html 
- test should be in spanish
- go to http://localhost:4000/components/monthview/test-specific-lang.html
- test should be in danish
- go to http://localhost:4000/components/datepicker/test-two-locales-same-page.html
- text will have different langs and locales in the same page
- go to http://localhost:4000/components/datepicker/test-specific-lang.html
- text should be spanish but the locale with dashes (danish)

**3464**
- go to http://localhost:4000/components/datepicker/example-index.html?locale=ru-RU
- open the picker and click the dropdown to see the months
- these are all changed compare to http://master-enterprise.demo.design.infor.com/components/datepicker/example-index.html?locale=ru-RU

**Included in this Pull Request**:
